rootfs/opt/ibm/router/nginx/template/nginx.tmpl

```diff
@@ -215,7 +215,7 @@ http {
         add_header X-Frame-Options "SAMEORIGIN";
         add_header X-Content-Type-Options nosniff;
         add_header X-XSS-Protection "1; mode=block";
-        add_header Strict-Transport-Security "max-age=31536000; includeSubDomains";
+        add_header Strict-Transport-Security "max-age=63072000; includeSubDomains";
```

Comment:
Is there an issue attached to this change? If so, we should link it within this PR.